README.md

```diff
@@ -112,7 +112,7 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **Swag**: Stickers (limit 40)
 - **Requirements**: Contribute code to the [Earthly](https://github.com/earthly/earthly) open source build tool or contribute an Earthfile build to any public repo.
 - **How to sign up**: [Grab an issue](https://github.com/earthly/earthly/issues?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest) and complete your PR after accepted please fill out this [form](https://forms.gle/DVkrWCu4yD2wMPmE8).
-- **Notes**: If you need any assistance, [join our slack channel](https://earthly.dev/slack). We are here to help! See our [Blog post](https://earthly.dev/blog/hacktoberfest-2021/) for more details.
+- **Notes**: If you need any assistance, [join thjeir Slack channel](https://earthly.dev/slack). See their [Blog post](https://earthly.dev/blog/hacktoberfest-2021/) for more details.
```

Comment:
thjeir Slack -> their Slack

Reply:
I swear I can type.... 😆 Fixing that now